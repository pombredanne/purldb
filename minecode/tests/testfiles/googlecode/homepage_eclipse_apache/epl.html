

<!DOCTYPE html>
<html>
<head>
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"token":null,"assetHostPath":"http://www.gstatic.com/codesite/ph","domainName":"eclipselabs.org","assetVersionPath":"http://www.gstatic.com/codesite/ph/1215397193597998757","projectName":"smebs","projectHomeUrl":"/a/eclipselabs.org/p/smebs","absoluteBaseUrl":"http://code.google.com/a/eclipselabs.org","relativeBaseUrl":"/a/eclipselabs.org","urlPrefix":"p","loggedInUserEmail":null};
 </script>
 
 
 <title>
 smebs -
 
 
 SMEBS - Top b2b  india | SME | b2b markeplace  - Google Project Hosting
 </title>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/1215397193597998757/css/ph_core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/1215397193597998757/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/1215397193597998757/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 
 .activity-level-High {
 background:no-repeat url(http://www.gstatic.com/codesite/ph/images/activity-level.png) 0 0;
 width: 16px;
 height: 16px;
 }
 .activity-level-Low {
 background:no-repeat url(http://www.gstatic.com/codesite/ph/images/activity-level.png) 0 -16px;
 width: 16px;
 height: 16px;
 }
 .activity-level-Medium {
 background: no-repeat url(http://www.gstatic.com/codesite/ph/images/activity-level.png) -16px 0;
 width: 16px;
 height: 16px;
 }
 .activity-level-None {
 background:no-repeat url(http://www.gstatic.com/codesite/ph/images/activity-level.png) -16px -16px;
 width: 16px;
 height: 16px;
 }


 </style>
</head>
<body class="t1">
 <script type="text/javascript">
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 </script>
<div class="headbg">
 <div id="gaia">
 

 <span>
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fa%2Feclipselabs.org%2Fp%2Fsmebs%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fa%2Feclipselabs.org%2Fp%2Fsmebs%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>
 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->




 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0">
 <tr style="height: 58px;">
 
 <td id="plogo">
 <a href="/a/eclipselabs.org/p/smebs/">
 
 
 <img src="/a/eclipselabs.org/p/smebs/logo?cct=1307210826" alt="Logo">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/a/eclipselabs.org/p/smebs/">smebs</a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link" href="/a/eclipselabs.org/p/smebs/" >SMEBS - Top b2b  india | SME | b2b markeplace </a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/a/eclipselabs.org/hosting/search">
 <input size="30" name="q" value="" type="text">
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/a/eclipselabs.org/p/smebs/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/a/eclipselabs.org/p/smebs/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/a/eclipselabs.org/p/smebs/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/a/eclipselabs.org/p/smebs/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/a/eclipselabs.org/p/smebs/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/a/eclipselabs.org/p/smebs/">Summary</a>
 </span>
 
 
 
 &nbsp;
 <span class="inst3">
 <a href="/a/eclipselabs.org/p/smebs/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 
<!-- IE -->





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 <ul class="pslist">
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://www.eclipse.org/legal/epl-v10.html" rel="nofollow">Eclipse Public License 1.0</a>
 
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:WebTools">WebTools</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:Build">Build</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:Mobile">Mobile</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:Modeling">Modeling</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:SOA">SOA</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:Tools">Tools</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:EclipseRT">EclipseRT</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:BIRT">BIRT</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:JavaEE">JavaEE</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:WebTools">WebTools</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:Embedded">Embedded</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:CodeManagement">CodeManagement</a>, 
 
 <a class="label"
 href="/a/eclipselabs.org/hosting/search?q=label:Testing">Testing</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <img height="15" width="15" src="http://www.gstatic.com/codesite/ph/images/people-y16.png"/>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 <a style="white-space: nowrap" href="/a/eclipselabs.org/u/ideosyncratic/">ideosyncratic</a>
 
 
 
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 <div class="phead pfeatured">Featured</div>
 
 <div class="psicon">
 <img height="15" width="15" src="http://www.gstatic.com/codesite/ph/images/filedownload-g16.png"/>
 <span><b>Downloads</b></span>
 </div>
 <ul class="pslist">
 
 <li id="downloadbox" class="psmeta">
 <a title="Download" href="//smebs.eclipselabs.org.codespot.com/files/Product%20or%20Service%20Type%20Menu.pdf" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//smebs.eclipselabs.org.codespot.com/files/Product%20or%20Service%20Type%20Menu.pdf', 'Featured, sme, B2B']);">
 </a>
 <a href="downloads/detail?name=Product or Service Type Menu.pdf"
 title="B2B Directory-product or service categories">Product or Service Type Menu.pdf</a>
 </li>
 
 <li class="psmeta"><a href="downloads/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 <div class="psicon">
 <img height="15" width="15" src="http://www.gstatic.com/codesite/ph/images/paper_pencil-y16.png"/>
 <span><b>Wiki pages</b></span>
 </div>
 <ul id="wikibox" class="pslist">
 
 <li class="psmeta">
 <a href="wiki/SMEBS" title=""
 >SMEBS</a>
 </li>
 
 <li class="psmeta"><a href="w/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 
 
 <div class="phead">Links</div>
 <ul class="pslist" style="white-space:nowrap">
 
 <li class="psmeta"><b>Blogs</b></li>
 
 <li class="psmeta">
 <a href="http://smebs.blog.com/" rel="nofollow">SMEBS-Top b2b blog for sme&#39;s india | small business blog | b2b market place blog</a>
 </li>
 
 <li class="psgap"/>
 
 
 <li class="psmeta"><b>External links</b></li>
 
 <li class="psmeta">
 <a href="http://www.sme.in/" rel="nofollow">Online B2B Directory</a>
 </li>
 
 <li class="psgap"/>
 
 
 </ul>
 
 </td>
 <td id="wikicontent" class="psdescription">
 <p>SMEBS Business to Business direct marketing blog specializing in lead generation, demand generation and customer retention. We deploy outbound calling campaigns on behalf of some of the nationâ€™s most well respected organizations. </p><p>We have become the direct marketing blog of choice for companies with complex products and services and best B2B-social media strategies being marketed to high level decision makers. Our direct marketing ideas help our clients increase sales through strategically designed lead generation programs. Whether the outbound calling program is to generate and qualify new prospects, enhance customer relationships, conduct marketing surveys or invite participants to events, sme is a creative and powerful B2B  partner that gets results. </p><p><a rel="nofollow" href="http://allthingsb2b.typepad.com/">B2b social marketing</a> <a rel="nofollow" href="http://www.smebs.blog.com/">B2b social media blog</a> </p>
 </td>
 </tr>
</table>
<script type="text/javascript" src="https://apis.google.com/js/plusone.js"></script>
<script src="http://www.gstatic.com/codesite/ph/1215397193597998757/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/1215397193597998757/js/ph_core.js"></script>
 
 
 
 
 <script type="text/javascript" src="/js/codesite_product_dictionary_ph.pack.04102009.js"></script>
</div> 
<div id="footer" dir="ltr">
 <div class="text">
 &copy;2011 Google -
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>
 
 


 
 </body>
</html>

