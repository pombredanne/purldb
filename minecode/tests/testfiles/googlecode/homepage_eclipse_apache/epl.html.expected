{'activity_level': 0,
 'blogs': None,
 'description': '<p>SMEBS Business to Business direct marketing blog specializing in lead generation, demand generation and customer retention. We deploy outbound calling campaigns on behalf of some of the nation\xe2\x80\x99s most well respected organizations. </p>',
 'download_page_url': None,
 'downloads': None,
 'external_links': None,
 'feeds': None,
 'forge': None,
 'groups': None,
 'homepage_url': u'http://code.google.com/a/eclipselabs.org/p/smebs/',
 'id': u'smebs',
 'labels': {u'BIRT': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:BIRT',
            u'Build': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:Build',
            u'CodeManagement': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:CodeManagement',
            u'EclipseRT': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:EclipseRT',
            u'Embedded': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:Embedded',
            u'JavaEE': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:JavaEE',
            u'Mobile': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:Mobile',
            u'Modeling': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:Modeling',
            u'SOA': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:SOA',
            u'Testing': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:Testing',
            u'Tools': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:Tools',
            u'WebTools': u'http://code.google.com/a/eclipselabs.org/hosting/search?q=label:WebTools'},
 'last_updated': None,
 'license': u'Eclipse Public License 1.0',
 'license_key': 'epl-1.0',
 'license_url': u'http://www.eclipse.org/legal/epl-v10.html',
 'logo_url': u'http://code.google.com/a/eclipselabs.org/p/smebs/logo?cct=1307210826',
 'members': None,
 'members_url': None,
 'name': u'SMEBS - Top b2b  india | SME | b2b markeplace ',
 'rank': None,
 'source_url': None,
 'stars': None,
 'summary_url': u'http://code.google.com/a/eclipselabs.org/p/smebs/',
 'update_url': None,
 'vcs_checkout_command': None}